e2e/fixtures/create-pages/src/components/DynamicLayout.tsx

```diff
@@ -1,15 +1,9 @@
 import type { ReactNode } from 'react';
 
-export default function DynamicLayout({
-  children,
-  path,
-}: {
-  children: ReactNode;
-  path: string;
-}) {
+export default function DynamicLayout({ children }: { children: ReactNode }) {
   return (
     <>
-      <nav>Current path: {path}</nav>
+      <nav>Current path: /dynamic</nav>
       {children}
     </>
   );
```

Comment:
I think this should use

```ts
const path = getContext().req.url.pathname;
```

instead of hard-coded `/dynamic`. 

It will show people how to migrate if they are using `path` in a dynamic layout.

Reply:
Yes, `req` is now a plain `Request`.